apps/dg/libraries/slickgrid/plugins/slick.rowselectionmodel.js

```diff
@@ -34,10 +34,12 @@
 
     function wrapHandler(handler) {
       return function () {
+        var result;
```

Comment:
There is a modification of slick.rowselectionmodel, but the case table installs its own model, DG.CaseTableRowSelectionModel.